indico/modules/categories/controllers/management.py

```diff
@@ -219,6 +220,26 @@ def _process(self):
         return jsonify_template('categories/management/move_category_contents.html', category=self.category)
 
 
+class RHDeleteSubcategories(RHManageCategoryBase):
+    """Bulk-delete subcategories"""
+
+    def _checkParams(self):
+        RHManageCategoryBase._checkParams(self)
+        self.subcategories = (Category.query.with_parent(self.category)
+                              .filter(Category.id.in_(request.args.getlist('category_id')))).all()
```

Comment:
I'd move the `.all()` and the `.with_parent(..)` into separate lines (i.e. one call per line)